gpep517/__main__.py

```diff
@@ -130,6 +130,22 @@ def install_wheel(args):
     return 0
 
 
+def verify_pyc(args):
+    from gpep517.qa import qa_verify_pyc
```

Comment:
I might be missing something, but where is this implemented?

Reply:
In the file I didn't add, obviously ;-).